src/Repository/RegionRepository.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Region;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+
+/**
+ * @extends ServiceEntityRepository<Region>
+ *
+ * @method Region|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Region|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Region[]    findAll()
+ * @method Region[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class RegionRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, Region::class);
+    }
+
+    /**
+     * @return bool Returns if you have exists region informed
+     */
+    public function findOneByIdExists(int $id): bool
+    {
+        $regionResult = $this->createQueryBuilder('region')
+            ->andWhere('region.id = :id')
+            ->setParameter('id', $id)
+            ->setMaxResults(1)
+            ->getQuery()
+            ->getResult();
+
+        return count($regionResult) > 0 && $regionResult[0]->getId() === $id;
```

Comment:
??????
`andWhere('region.id = :id')
`

Why would you check if the id from the result is === $id since you're already doing it in the SQL query ?

Just return the region result if it exists